src/lib/plugin.js

```diff
@@ -134,37 +124,58 @@ module.exports = class PluginVirtual extends EventEmitter2 {
     }
   }
 
-  * _connect () {
+  registerRequestHandler (handler) {
+    if (typeof handler !== 'function') {
+      throw new InvalidFieldsError('requestHandler must be a function')
+    }
```

Comment:
If `this._requestHandler` is already set, `registerRequestHandler()` should throw a `RequestHandlerAlreadyRegisteredError` (see https://github.com/interledger/rfcs/blob/e4dab0259283a500f923f92ca4f50b18509cdc44/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#registerrequesthandler).